home/views/api/leaderboard.py

```diff
@@ -0,0 +1,365 @@
+# from datetime import date, timedelta
+# from typing import Optional
+import json
+
+
+from django.http import JsonResponse
+from django.utils.decorators import method_decorator
+from django.views import View
+from django.views.decorators.csrf import csrf_exempt
+
+# from django.db.models import Count, F, Sum
+from django.core.exceptions import ObjectDoesNotExist
+
+
+from home.models import (
+    # Account,
+    Contest,
+    # DailyWalk,
+    # IntentionalWalk,
+    Device,
+    Leaderboard,
+)
+
+# from home.utils import localize
+from .utils import validate_request_json
+
+
+DEVICE_FIELDS = [
+    "account",
+    "account_id",
+    "device_id",
+]
+ACCOUNT_FIELDS = [
+    "email",
+    "name",
+    "age",
+    "zip",
+    "is_tester",
+    "created",
+    "gender",
+    "gender_other",
+    "race",
+    "race_other",
+    "is_latino",
+    "sexual_orien",
+    "sexual_orien_other",
+    "id",
+    # "device",
+]
+
+
+# def get_daily_walk_summaries(**filters):
```

Comment:
I'd just delete all this commented code- if you need to refer to it, you can look in the other file...?